contracts/CO30_Emerson_RedSkyAtNight.Tanoa/scripts/transcript.sqf

```diff
@@ -0,0 +1,9 @@
+params ["_transcript"];
+
+_player = player;
+
+_timeString = [dayTime, "HH:MM"] call BIS_fnc_timeToString;
+_transcriptTitle = "Transmission from " + _timeString;
+
+_player createDiaryRecord ["Diary", [_transcriptTitle, _transcript]];
+true
```

Comment:
```suggestion
params ["_transcript"];

player createDiaryRecord ["Diary", [
    "Transmission from " + ([dayTime, "HH:MM"] call BIS_fnc_timeToString),
    _transcript
]];
```
the variables can easily be avoided since they are used once